Dshop/apps/products_catalogue/views.py

```diff
@@ -64,22 +64,34 @@ def generate_xml_file_for_ceneo(products):
 
         for product in products:
             o_element = etree.SubElement(root, 'o', id=str(product.pk), url=str(), price=str(product.price),
-                                         avail=str(),   # TODO: url and available items
+                                         avail=str(product.availability),  # TODO: add url
                                          weight=str(),
                                          stock=str(),
-                                         basket=str())  
+                                         basket=str())
 
             cat_element = etree.SubElement(o_element, 'cat')
             cat_element.text = etree.CDATA(product.category.name)
 
             name_element = etree.SubElement(o_element, 'name')
             name_element.text = etree.CDATA(product.name)
 
+            if product.featured_photos.exists():
+                featured_image_url = product.featured_photos.first().image.url
+                print(featured_image_url)
```

Comment:
Please remove print

Reply:
Done, sorry for this. 
But I have a problem with migrations now: ModuleNotFoundError: No module named 'dj_shop_cart'
Tests didn't pass.

Reply:
Solved. I had to install it manually.

Reply:
What do you mean manually?

Reply:
Pip install dj... cart and pip install charset-normalizer. I had errors using poetry update.